libbeat/docs/processors-using.asciidoc

```diff
@@ -511,15 +511,18 @@ processors:
 - add_cloud_metadata: ~
 -------------------------------------------------------------------------------
 
-The `add_cloud_metadata` processor has one optional configuration setting named
-`timeout` that specifies the maximum amount of time to wait for a successful
+The `add_cloud_metadata` processor has two optional configuration settings. First one is
```

Comment:
```suggestion
The `add_cloud_metadata` processor has two optional configuration settings. The first one is
```

Reply:
Most of the lines in this file wrap a 80 chars. It would nice to keep format. My edits don't reflect this format since the suggestions are limited to one line.

Reply:
Good to know, i just set my ide margin to 80 👍 Thanks!